site/app/models/Config.php

```diff
@@ -24,7 +24,7 @@ class Config {
      * turn on if running server in production environment.
      * @var bool
      */
-    private $debug = __INSTALL__FILLIN__DEBUGGING_ENABLED__;
+    private $debug = false;
```

Comment:
Why? If we were set it in configure/install why should we have a debug flag in the master.ini file (hint: it'd be redundant, so why bother with master.ini at all? answer: because not bothering with a configuration file is bad design).

There's no harm in setting this to false by default and makes it more explicit as to you have to turn on debug in master.ini to have debug on (as otherwise you could end up with the scenario where you turn on debug in configure/install, remove the debug line completely from master.ini and then still have debug enabled which is probably not desired)